### ISSUE TITLE
Remove gh-page specific files from master

### ISSUE THREAD

**Issue by kristoferjoseph** [kristoferjoseph/flexboxgrid#73]
Currently we add all of the files needed to for the gh-page as well as the grid in the master branch. After seeing a few projects that accidentally include the flexboxgrid.com code I am thinking only having the grid code in master is a better idea. 

Since I have been the only person updating the site it makes sense to just pull in PRs by hand.

**Comment by dandv** [comment]
Some have chosen to [use the `gh-pages` branch as the master branch](http://stackoverflow.com/questions/15285324/is-there-a-way-to-use-a-pre-post-commit-hook-in-github-to-update-gh-pages-branch/30646245#30646245).

**Comment by Aybee** [comment]
I'm not happy with the demo. The index.html is to complicated - e.g. loading CSS ressource with JS. Mixing a normalize, multiple custom and a minified flexbox-grid CSS into one file. 

The src/index.html has no CSS at all.

My proposal is to offer an index.html that simply loads an uncompressed flexbox-grid and separately some other CSS for beautifying. No JS. And all CLASS attributes that do not belong to flexboxgrid should be removed so we can see what belongs to flexboxgrid. Maybe the boxes can contain a small text content.

In short:
- No JS
- Separate CSS files
- Only flexboxgrid classes
- Maybe a small text in the boxes

**Comment by kristoferjoseph** [comment]
@Aybee I hear what you are saying. The site is built the way it is in order to load as fast as possible. Check [page speed insights](https://developers.google.com/speed/pagespeed/insights/) if you would like to learn about these optimizations. With that said I am planning on making a repo with just flexboxgrid examples that will be exactly what you are requesting.

**Comment by Aybee** [comment]
Yes I know about page speed. But I think on a demo page it's more important to get a quick approach to the essentials and the possibility of easy fiddling around.

**Comment by dandv** [comment]
:+1: for optimizing for legibility.

**Comment by kristoferjoseph** [comment]
@aybee Your confusion here is that flexboxgrid is a marketing page not a demo page. I feel code pens and example projects will always be better suited for the use case you are describing. 

@dandv are you having a hard time reading the page? Or are you having a hard time understanding it? Optimizing for legibility is probably not what you mean to say here.

**Comment by Aybee** [comment]
Sorry, but I don't have the time to do it right now and I'm less motivated as I am not using the grid right now. But I think it will be enough if there is a sample (demo) page inside the download that is as I described above.

**Comment by kristoferjoseph** [comment]
:+1:

**Comment by kristoferjoseph** [comment]
@Aybee the "good new" is eventually this won't be needed http://bit.ly/1I1YJzy

Until then we have to live with the web as a compile target, which I agree, is gross.

**Comment by kristoferjoseph** [comment]
Removing the gh-pages branch will clear up some confusion as well. People ask whether files used solely for the website are bundled with the grid for instance normalize.css. The answer is they are only being used by the website and only bits specific to the grid are included in the npm package and cdn link.
